### ISSUE TITLE
Built-in function for array.except?

### ISSUE THREAD

**Issue by joshidp** [scriban/scriban#505]
Is there a Built-in function for array.except? It needs takes two arrays arrA and arrB, returns values which are in arrA and not in arrB.

**Comment by xoofx** [comment]
Nope, but you can always add one on your side. It should be almost a one liner.
